test/test_profiler_tree.py

```diff
@@ -316,34 +316,18 @@ def test_profiler_experimental_tree_with_record_function(self):
         self.assertTreesMatch(
             ProfilerTree.format(p.profiler, 12),
             """\
-            aten::zeros
-              aten::empty
-              aten::zero_
```

Comment:
This is expected because the `aten::zeros` call was coming from `record_function.__init__`.